Formula/mariadb@10.8.rb

```diff
@@ -36,7 +36,9 @@ class MariadbAT108 < Formula
   keg_only :versioned_formula
 
   # See: https://mariadb.com/kb/en/changes-improvements-in-mariadb-108/
-  deprecate! date: "2023-05-01", because: :unsupported
+  # and https://mariadb.org/about/#maintenance-policy
+  # End-of-life on 2023-05-20. Disable date set to 3 months after.
+  disable! date: "2023-08-20", because: :unsupported
```

Comment:
Updated to use official EOL date from https://mariadb.org/about/#maintenance-policy +3 months to disable.

Release series | Stable (GA) Date | End-of-life
--- | --- | ---
[10.8](https://mariadb.com/kb/en/changes-improvements-in-mariadb-108/) | 20 May 2022 | **20 May 2023**

Reply:
Similar to other PRs, switching back to `deprecate!` to avoid showing up as deprecated when it is not. Following is output with `disable!`:
```console
❯ brew info mariadb@10.8
==> mariadb@10.8: stable 10.8.7 (bottled) [keg-only]
Drop-in replacement for MySQL
https://mariadb.org/
Deprecated because it is not supported upstream!
```

Unless the message is adjusted in `brew`, it doesn't accurately represent state of formula.